app/services/work_packages/set_schedule_service.rb

```diff
@@ -174,16 +174,14 @@ def schedule_on_missing_dates(scheduled, min_start_date)
               scheduled.due_date && scheduled.due_date < min_start_date ? min_start_date : scheduled.due_date)
   end
 
-  def reschedule_by_delta(scheduled, moved_delta, min_start_date, dependency)
-    days = WorkPackages::Shared::Days.for(dependency.work_package)
+  def reschedule_by_delta(scheduled, moved_delta, min_start_date)
+    days = WorkPackages::Shared::Days.for(scheduled)
```

Comment:
At first sight this change seems to be unrelated to removing the flag. Is this intended to be part of the PR?

Reply:
True, this is not related. That's something I could have done in another PR. Should I remove it?

Reply:
If it fails no tests, I'm fine with it. Perhaps do it in a separate commit?

Reply:
I just removed it from this PR. I'll add it in my other PR. Thanks for the feedback.